lib/std/io/tty.zig

```diff
@@ -133,3 +133,43 @@ pub const Config = union(enum) {
         };
     }
 };
+
+/// Registers a global handler function to run if an interrupt signal is catched.
+/// An interrupt signal is usually fired if Ctrl+C is pressed in the terminal that controls the process.
+///
+/// Because this handler won't run if Ctrl+C isn't pressed by the user, registering this handler failed,
+/// or the handler was overwritten by another setInterruptSignalHandler call,
+/// this should be used only for non-critical cleanups or resets of terminal state and such.
+///
+/// A program can only have one handler at a time.
+///
+/// The handler will not exit the program after it runs.
+pub fn setInterruptSignalHandler(comptime handler: *const fn () void) error{Unexpected}!void {
```

Comment:
If the `handler` is comptime, then is there any reason we're using a function pointer instead of a function body here?

Reply:
You mean `fn () void`?

Reply:
Yes, exactly. Not a rhetorical question btw, I genuinely don't know if you had a reason.

Reply:
Nah good point. No reason to make this a function pointer.